tests/h/schemas/admin_group_test.py

```diff
@@ -124,13 +120,20 @@ def user_validator(self, user_svc):
 
 @pytest.fixture
 def group_data(factories):
+    """
+    Return a serialized representation of the "Create Group" form.
+
+    This is the representation that Deform passes to Colander for
+    deserialization and validation after the HTML form is processed by
+    Peppercorn.
+    """
```

Comment:
I added this since it is not that obvious what the relation of this fixture is to the data that actually gets submitted by. the user's browser when they save the form.